packages/packages/icons/src/components/wide-screen-icon.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { SvgIcon, SvgIconProps } from '@elementor/ui';
+
+const WideScreenIcon = React.forwardRef( ( props: SvgIconProps, ref ) => {
```

Comment:
https://dictionary.cambridge.org/dictionary/english/widescreen

it's a single word 🤷

Reply:
Done.